src/main/java/com/example/bookstore/repository/ShoppingCartRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstore.repository;
+
+import com.example.bookstore.model.ShoppingCart;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
+    @Query("select cart from ShoppingCart cart "
```

Comment:
You can use String blocks for such long concatenations to make it easier to read and mantain.